scripts/build_npm.ts

```diff
@@ -1,6 +1,23 @@
 import { build, emptyDir } from "https://deno.land/x/dnt/mod.ts";
+import { copy } from "https://deno.land/std@0.177.0/fs/mod.ts";
 
-await emptyDir("./npm");
+// Copy test data
+await Deno.remove("npm", { recursive: true }).catch((_) => {});
+await emptyDir("./npm/esm/src/m3/ecore/test");
+await emptyDir("./npm/script/src/m3/ecore/test");
+await emptyDir("./npm/esm/src/m3/test");
+await emptyDir("./npm/script/src/m3/test");
+
+await copy("diagrams", "npm/esm/diagrams", { overwrite: true });
+await copy("diagrams", "npm/script/diagrams", { overwrite: true });
+await copy("models", "npm/esm/models", { overwrite: true });
+await copy("models", "npm/script/models", { overwrite: true });
+await copy("schemas", "npm/esm/schemas", { overwrite: true });
+await copy("schemas", "npm/script/schemas", { overwrite: true });
+await copy("src/m3/ecore/test/library.ecore", "npm/esm/src/m3/ecore/test/library.ecore", { overwrite: true });
```

Comment:
I think all test-artifacts shouldn't end up in `npm/`.

Reply:
I would think so, too. But I don't know Deno's conventions, and seeing that they are part of the source and the "dnt" tool explicitly runs them and finds them where they are, I assumed that's standard practice for Deno packages. I think we can safely remove them after the build step.

Reply:
OK, we'll look how it works exactly after merging ;)